docs/core/scene.md

```diff
@@ -52,7 +52,7 @@ The scene inherits from the [`Entity`][entity] class so it inherits all of its p
 | enter-vr     | User has entered VR and headset started presenting content. |
 | exit-vr      | User has exited VR and headset stopped presenting content.  |
 | loaded       | All nodes have loaded.                             |
-| render-start | Render loop has started.            |
+| renderStarted| Render loop has started.            |
```

Comment:
I have a swoop at https://github.com/aframevr/aframe/pull/1494 , but please PR if you see anything, git will handle the merges :)